ui/cypress/support/helpers.js

```diff
@@ -93,24 +121,25 @@ export async function waitForAllUrlsToFinish(urls) {
 // NOTE use cy.wrap on tests
 export async function waitUrlToProcess(url, property, maxTries = 6, method='GET') {
 
-  let response = await p(cy.request({
+  let response = await axiosClient({
     method,
     url,
-    failOnStatusCode: false
-  }));
+    failOnStatusCode: false,
+    auth
+  });
 
   let tries = 0;
 
   while (!get(response, property) && tries < 5) {
     tries++;
-    console.log('url', url, property, 'did not finish', 'tries:', tries, '. Retrying.');
-    cy.wait(1000);
+    await sleep(1000);
 
-    let response = await p(cy.request({
```

Comment:
This ugly `p()` wrapper around Cypress promises is mostly a hack- a third-party library in order to use `async/await` on Cypress promises. Did away from that here and we're now using plain old real Promises on these helper fns.